index.html

```diff
@@ -17,12 +17,12 @@
 Schedule
 --------
 
-* 7:00 PM -- Introductions
-* 7:10 PM -- First speaker · [Submit Talk][4]
-* 7:25 PM -- Second speaker · [Submit Talk][4]
-* 7:40 PM -- Third speaker · [Submit Talk][4]
-* 7:55 PM -- Fourth speaker · [Submit Talk][4]
-* 8:10 PM -- Hang out
+* <span class="time">7:00 PM</span> -- Introductions
+* <span class="time">7:10 PM</span> -- First speaker · [Submit Talk][4]
+* <span class="time">7:25 PM</span> -- Second speaker · [Submit Talk][4]
+* <span class="time">7:40 PM</span> -- Third speaker · [Submit Talk][4]
+* <span class="time">7:55 PM</span> -- Fourth speaker · [Submit Talk][4]
+* <span class="time">8:10 PM</span> -- Hang out
```

Comment:
I'd do this with bold or italics (`**` or `*` or `__` or `_`) and style using that.